docs/life_of_a_godwoken_transaction.md

```diff
@@ -327,7 +314,7 @@ Here is the JSON representation for a layer 2 transaction, which invokes MetaCon
 }
 ```
 
-The `args` part in this transaction, contains a [MetaContractArgs](https://github.com/nervosnetwork/godwoken/blob/v0.1.0/crates/types/schemas/godwoken.mol#L192-L194) data structure serialized in molecule format. A JSON representation for the `args`, is as follows:
+The `args` part in this transaction, contains a [MetaContractArgs](https://github.com/nervosnetwork/godwoken/blob/v1.1.0-beta/crates/types/schemas/godwoken.mol#L232) data structure serialized in molecule format. A JSON representation for the `args`, is as follows:
```

Comment:
The `args` part of this transaction contains a [MetaContractArgs](https://github.com/nervosnetwork/godwoken/blob/v1.1.0-beta/crates/types/schemas/godwoken.mol#L232) data structure serialized in the molecule format. A JSON representation of the `args` is as follows: